### DISCUSSION TITLE
贼离谱啊

### DISCUSSION THREAD

**Discussion by ZS2147483647** [Meloong-Git/PCL#3309] [提问] [answered]
我和室友联机,1.19.2,他说报错,然后一问说用的是jre8,然后我告诉他用jdk17,他居然说jre8启动成功了为什么
怎么会一会成功一会不行呢,jre8到底能不能启动高版本

**Comment by ZS2147483647** [comment]
我是一次没成功过

**Comment by zkitefly** [comment]
不能

**Comment by allMagicNB** [comment]
不能，也许他当时也不知道用的啥Java

**Comment by MoYuan-CN** [comment]
不能，有可能 PCL 自动选择了合适的 Java

**Comment by WForst-Breeze** [comment, marked as answer]
都报错了能叫启动成功了吗🤯

**Reply by MoYuan-CN** [reply]
论 `启动成功` 的定义

**Reply by allMagicNB** [reply]
论“我上次用它明明成功了”的“它”指代的是什么

**Reply by ZS2147483647** [reply]
是这样的，他没换Java（就用jre8）就成功进入游戏并和我联机玩了

**Reply by allMagicNB** [reply]
> 是这样的，他没换Java（就用jre8）就成功进入游戏并和我联机玩了

有证据？给日志！

**Comment by burningtnt** [comment]
不给错误报告 = 闭眼开车

你如果真要求助，给错误报告

**Reply by ZS2147483647** [reply]
我问的是Java，而且别人玩的哪来的错误报告

**Reply by allMagicNB** [reply]
> 我问的是Java，而且别人玩的哪来的错误报告

那就给日志呗

**Reply by ZS2147483647** [reply]
别人玩的我这没日志

**Comment by ZS2147483647** [comment]
![mmexport1707643212592](https://github.com/Hex-Dragon/PCL2/assets/121028009/70e5ce67-70bc-4ff2-85ff-216a89ff3af9)
他当时的截图

**Reply by allMagicNB** [reply]
我们要的是JRE8启动1.19.2成功的日志
